features/default/configuration.feature

```diff
@@ -24,4 +24,32 @@ Feature: Configuration overrides
     * the trace payload field "resourceSpans.0.resource" string attribute "service.name" equals "com.bugsnag.ServiceNameScenario"
     * the trace payload field "resourceSpans.0.resource" string attribute "service.version" equals "10.0"
     * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.name" equals "bugsnag.performance.cocoa"
-    * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.version" matches the regex "[0-9]\.[0-9]\.[0-9]"
\ No newline at end of file
+    * the trace payload field "resourceSpans.0.resource" string attribute "telemetry.sdk.version" matches the regex "[0-9]\.[0-9]\.[0-9]"
+
+  Scenario: Fixed sampling probability
```

Comment:
I just left a similar comment on the Unity equivalent of this test – as the step is now a little confusingly named (now we have "sampling probability" as a config option), can you expand the scenario text to clarify?

> Setting fixed sampling probability of 1 with dynamic probability of 0 should send all spans

Reply:
https://github.com/bugsnag/bugsnag-unity-performance/pull/128/files#r1724865481